chain/chain/src/chain.rs

```diff
@@ -4416,6 +4512,7 @@ impl Chain {
     }
 
     /// Check that sync_hash is the first block of an epoch.
+    /// TODO: allow the new way of computing the sync hash for syncing to the current epoch
```

Comment:
nit: for all todos can you either add a unique name or an issue id? e.g. TODO(current_epoch_state_sync)

Reply:
done